relayer-cli/build.rs

```diff
@@ -54,24 +52,6 @@ mod git {
             }
         }
 
-        // Returns the current branch as a string - if the branch name has a '/' returns part after
-        // the last '/'
-        pub fn branch(&self) -> String {
-            let branch = Self::command(&["rev-parse", "--abbrev-ref", "HEAD"]);
-            let branch_str = String::from_utf8_lossy(&branch.stdout);
-            branch_str
```

Comment:
An alternative would be to keep the branch name but replace all invalid chars (i.e. chars other than alphanumeric, dots and hyphens), something like ->
```rust
branch_str.replace(|c: char| !c.is_ascii_alphanumeric(), "");
```

Reply:
I thought about that but in the end I'd rather just omit it, rather than include a garbled version. The branch can be inferred from the commit anyway for those who are developing the software, and branch names don't carry that much useful information anyway for people who are not devs. Does that make sense?

Reply:
Right, makes sense. 👍